README.md

```diff
@@ -1,12 +1,18 @@
 # ember-osf-web
 
-[![Greenkeeper badge](https://badges.greenkeeper.io/CenterForOpenScience/ember-osf-web.svg)](https://greenkeeper.io/)
-
-`master` Build Status: [![Build Status](https://travis-ci.org/CenterForOpenScience/ember-osf-web.svg?branch=master)](https://travis-ci.org/CenterForOpenScience/ember-osf-web)
-[![Coverage Status](https://coveralls.io/repos/github/CenterForOpenScience/ember-osf-web/badge.svg?branch=master)](https://coveralls.io/github/CenterForOpenScience/ember-osf-web?branch=master)
-
-`develop` Build Status: [![Build Status](https://travis-ci.org/CenterForOpenScience/ember-osf-web.svg?branch=develop)](https://travis-ci.org/CenterForOpenScience/ember-osf-web)
-[![Coverage Status](https://coveralls.io/repos/github/CenterForOpenScience/ember-osf-web/badge.svg?branch=develop)](https://coveralls.io/github/CenterForOpenScience/ember-osf-web?branch=develop)
+`master build`<a href="https://github.com/CenterForOpenScience/ember-osf-web/actions">
+    <img alt="master build status" style="margin-bottom: -4px" src="https://github.com/CenterForOpenScience/ember-osf-web/workflows/CI/badge.svg?branch=master">
+</a>
+<a href="https://coveralls.io/github/CenterForOpenScience/ember-osf-web?branch=master">
+    <img alt="Coverage Status" style="margin-bottom: -4px;" src="https://coveralls.io/repos/github/CenterForOpenScience/ember-osf-web/badge.svg?branch=master">
+</a>
+
+`develop build`<a href="https://github.com/CenterForOpenScience/ember-osf-web/actions">
+    <img alt="develop build status" style="margin-bottom: -4px;" src="https://github.com/CenterForOpenScience/ember-osf-web/workflows/CI/badge.svg?branch=develop">
+</a>
+<a href="https://coveralls.io/github/CenterForOpenScience/ember-osf-web?branch=master">
+    <img alt="Coverage Status" style="margin-bottom: -4px;" src="https://coveralls.io/repos/github/CenterForOpenScience/ember-osf-web/badge.svg?branch=develop">
+</a>
```

Comment:
went with html here instead of markdown because the text (`develop build`) and status badge are not aligned properly.